module/optee_mhu/include/mod_optee_mhu.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Arm SCP/MCP Software
+ * Copyright (c) 2022, Linaro Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Description:
+ *     OP-TEE mailbox buffer layer
+ */
+
+#ifndef MOD_OPTEE_MHU_H
+#define MOD_OPTEE_MHU_H
+
+/* OPTEE_MHU currently uses a static shared memory buffer size */
+#define OPTEE_MHU_SHMEM_SIZE 128
+
+/*!
+ * \brief Channel type
+ *
+ * \details Defines the role of an entity in a channel
+ */
+enum mod_optee_mhu_channel_type {
+    /*! Master channel */
+    MOD_OPTEE_MHU_CHANNEL_TYPE_MASTER,
+
+    /*! Slave channel */
+    MOD_OPTEE_MHU_CHANNEL_TYPE_SLAVE,
```

Comment:
This is a deprecated term as it doesn't conform to our inclusive language policy